<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UImageConvertationForm</class>
 <widget class="QMainWindow" name="UImageConvertationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UImageConvertationForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lEditUrlPicture">
          <property name="toolTip">
           <string>Поле ввода адреса изображения</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&amp;Адрес:</string>
          </property>
          <property name="buddy">
           <cstring>lEditUrlPicture</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tButtonSelectFile">
          <property name="toolTip">
           <string>Выбрать изображение...</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tButtonQuit">
          <property name="toolTip">
           <string>Выход</string>
          </property>
          <property name="text">
           <string>Выход</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/IconQuit</normaloff>:/IconQuit</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tButtonSaveAs">
          <property name="toolTip">
           <string>Сохранить в...</string>
          </property>
          <property name="text">
           <string>Сохранить в...</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/IconSave</normaloff>:/IconSave</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tButtonDownloadPicture">
          <property name="toolTip">
           <string>Загрузить</string>
          </property>
          <property name="text">
           <string>Загрузить</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/IconDownload</normaloff>:/IconDownload</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tButtonAbortDownloadPicture">
          <property name="toolTip">
           <string>Прервать</string>
          </property>
          <property name="text">
           <string>Прервать</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/IconAbort</normaloff>:/IconAbort</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>401</width>
             <height>156</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="labelImage">
              <property name="acceptDrops">
               <bool>true</bool>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetProgressBar" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QProgressBar" name="progressBarDownload">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>13</height>
              </size>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="textDirection">
              <enum>QProgressBar::TopToBottom</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelErrors">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QListWidget" name="lWidgetImages">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerItem</enum>
       </property>
       <property name="isWrapping" stdset="0">
        <bool>false</bool>
       </property>
       <property name="resizeMode">
        <enum>QListView::Fixed</enum>
       </property>
       <property name="layoutMode">
        <enum>QListView::SinglePass</enum>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="viewMode">
        <enum>QListView::ListMode</enum>
       </property>
       <property name="modelColumn">
        <number>0</number>
       </property>
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>677</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>&amp;Справка</string>
    </property>
    <addaction name="actionAboutProgram"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>&amp;Изображение</string>
    </property>
    <addaction name="actionSelectFile"/>
    <addaction name="actionSelectFolder"/>
    <addaction name="separator"/>
    <addaction name="actionDownloadPicture"/>
    <addaction name="actionAbortDownloadPicture"/>
    <addaction name="separator"/>
    <addaction name="actionSaveAs"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_4">
   <property name="windowTitle">
    <string>toolBar_4</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionAboutProgram">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/IconQuit</normaloff>:/IconQuit</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/IconSave</normaloff>:/IconSave</iconset>
   </property>
   <property name="text">
    <string>Сохранить в...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSelectFile">
   <property name="text">
    <string>Выбрать изображение...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDownloadPicture">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/IconDownload</normaloff>:/IconDownload</iconset>
   </property>
   <property name="text">
    <string>Загрузить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionAbortDownloadPicture">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/IconAbort</normaloff>:/IconAbort</iconset>
   </property>
   <property name="text">
    <string>Прервать загрузку</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSelectFolder">
   <property name="text">
    <string>Выбрать папку...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
